h/routes.py

```diff
@@ -64,6 +64,7 @@ def includeme(config):
     config.add_route('badge', '/api/badge')
     config.add_route('token', '/api/token')
     config.add_route('api.users', '/api/users')
+    config.add_route('api.profile', '/api/profile')
```

Comment:
We usually keep things ordered alphabetically (@nickstenning will always leave a comment when he sees something that isn't correctly ordered 😉). Although this block of code is already in a bad order, so not really your fault at all 🙂